aframe/src/networking.js

```diff
@@ -1,32 +1,61 @@
 import {log} from './logger';
+import axios from 'axios';
 import uuidv4 from 'uuid/v4';
 
 // Modify to test a local server
 // const API_BASE = 'http://localhost:3000/1.0';
 
 // TODO: Need to change the API base The Graph to fetch correct ad
-const API_BASE = null;
-const AD_ENDPOINT = API_BASE + '/ad';
+const AD_ENDPOINT = 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-rinkeby'
```

Comment:
We should have a config where we can easily point to multiple different networks.